appconfiguration/client/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -44,7 +45,8 @@
  * @see ConfigurationAsyncClientBuilder
  * @see ConfigurationClientCredentials
  */
-public final class ConfigurationAsyncClient extends ServiceClient {
+@ServiceClient(builder = ConfigurationAsyncClientBuilder.class)
+public final class ConfigurationAsyncClient {
     private static final String SPAN_NAME_TEMPLATE = "Azure.AppConfig/%s";
```

Comment:
That would be my hope. I haven't gone into tracing-specific investigations this time around, but I'll explore at some stage. There is the slight pain that the tracing information needs to be on the service interface rather than the service client, but we will see....